comtypes/_memberspec.py

```diff
@@ -175,14 +175,15 @@ def COMMETHOD(idlflags, restype, methodname, *argspec) -> _ComMemberSpec:
     # join them together(does this make sense?) and replace by None if empty.
     helptext = "".join(t for t in idlflags if isinstance(t, helpstring)) or None
     paramflags, argtypes = _resolve_argspec(argspec)
-    if "propget" in idlflags:
-        name = "_get_%s" % methodname
-    elif "propput" in idlflags:
-        name = "_set_%s" % methodname
-    elif "propputref" in idlflags:
-        name = "_setref_%s" % methodname
-    else:
-        name = methodname
+    name = (
+        f"_get_{methodname}"
+        if "propget" in idlflags
+        else f"_set_{methodname}"
+        if "propput" in idlflags
+        else f"_setref_{methodname}"
+        if "propputref" in idlflags
+        else methodname
+    )
```

Comment:
If [conditional expressions (aka "ternary operators")](https://docs.python.org/3/reference/expressions.html#conditional-expressions) becomes long enough that it is broken into multiple lines by the formatter, it is likely more readable to use regular [`if` statements](https://docs.python.org/3/tutorial/controlflow.html#if-statements) instead.